src/util.h

```diff
@@ -48,6 +48,13 @@ extern bool fLogIPs;
 extern volatile bool fReopenDebugLog;
 extern CTranslationInterface translationInterface;
 
+static const char BITCOIN_CONF_FILENAME[] = "bitcoin.conf";
```

Comment:
So please move the content of the variable and `BITCOIN_PID_FILENAME` to util.cpp
Apart from that I'd like to merge this